src/components/atoms/FavoriteCount/FavoriteCount.tsx

```diff
@@ -0,0 +1,20 @@
+// import { Inter } from "next/font/google"
+import Image from "next/image"
+
+interface FavoriteCountProps {
+  likeIcon: string
+  count: number
+}
+
+export const FavoriteCount = (props: FavoriteCountProps) => {
+  return (
+    <>
+      {/* 親要素にrelativeを設定してくだい */}
+      <div className="absolute bg-Mauve-alpha-10 w-[67px] h-[26px] rounded-2xl opacity-50"></div>
+      <div className="absolute  h-[14px] flex w-[55px] leading-[14px] p-[6px]">
+        <Image src={props.likeIcon} alt="お気に入り数" width={14} height={14} className="h-[14px]" />
+        <div className=" text-Mauve-01 text-fs14 font-inter">{props.count.toLocaleString()}</div>
+      </div>
+    </>
```

Comment:
テキストで全部伝えるの難しそうかも？と思ったので、
こんな感じかなー？というのを載せときます！
topとrightも固定で入れておいてもいいと思いますー！

```suggestion
<div className="absolute top-2 right-2 flex p-[6px] bg-Mauve-alpha-10 rounded-2xl opacity-50">
  <Image src="heart.svg" alt="お気に入り数" width={14} height={14} />
  <p className="text-Mauve-01 text-fs14 leading-none ml-1 font-inter">{props.count.toLocaleString()}</p>
</div>
```
ざっくり対応内容
1. 全部divになっているのを修正
2. 不要なelementを削除（React.Fragment, div）
3. 不要なスタイル削除
5. その他スタイル調整

Reply:
とても分かりやすくありがとうございます✨
ご指摘の箇所修正させて頂きます✨🙏

一点上記の中で一番上のdivタグで記載していた箇所なのですが、当初こちらでopacityを設定すると、
tailwindの仕様上配下のすべてに透過が上書きされてしまい、
ここでいうと数字の色等などが全てに対して透過されてしまい、見にくくなってしまうため、
数値等が透過されないように無理やりdivタグにて透過した背景を作成して右側のようにしていたのですが、
記載について良い方法ありますでしょうか？

（上記の修正いただいたコードだと左の形で数字まで透過されてしまうようです🙇‍♂️）

<img width="284" alt="スクリーンショット 2023-06-26 21 05 22" src="https://github.com/qin-team-recipe/07-recipe-app-front/assets/73697917/0384aa64-5715-4c63-bf92-d6e4c0dd93fd">
<img width="404" alt="スクリーンショット 2023-06-26 21 06 19" src="https://github.com/qin-team-recipe/07-recipe-app-front/assets/73697917/98a93fa6-adca-4505-9d19-2781bd2dbb31">

Reply:
@kyoichi1 
なるほどですー！！
`opacity-50`ではなく、`bg-opacity-50`使うとよさそうです 👍 

ショートハンドで`bg-Mauve-alpha-10/50`とも書けそうです！（背景色/透過度）
https://tailwindcss.com/docs/background-color#changing-the-opacity

Reply:
@yuktnk 
ありがとうございます😭🙏✨
bgだけ透過させるbg-opacityがあるんですね👀！
すみません勉強&調査不足でした🙇‍♂️💦
bg-opacity-50の方に修正させて頂きます☺️✨